server-phoenix/lib/helios_web/schema/resolvers/widget_collection.ex

```diff
@@ -1,22 +1,18 @@
 defmodule HeliosWeb.Schema.Resolvers.WidgetCollection do
-  alias HeliosWeb.TestData.WidgetCollection, as: WidgetCollectionData
-  def enabled(_parent, _args, _info) do
-    {:ok, Jason.decode!(WidgetCollectionData.enabled)}
+  alias HeliosWeb.Schema.Helpers.WidgetCollection, as: WidgetCollectionHelpers
+  import Ecto.Query
+  alias Helios.{Repo, Widget}
+
+  def enabled(parent, _args, _info) do
```

Comment:
Same question here. Can we assert that parent is `Location`. Something like:
`def enabled(parent, _args, _info) when is_struct(parent, Location) do`